docs/snippets/solid/list-story-with-unchecked-children.js.mdx

```diff
@@ -0,0 +1,23 @@
+```js
```

Comment:
@webblocksapp one question about this snippet and the TS variations. I was looking at this and compared it with your reproduction [here](https://github.com/webblocksapp/solidjs-sb-sandbox/blob/master/src/stories/ListItemSelections.stories.tsx), and they don't match, to give you a bit more context that snippet is only featured for React as it's possible to apply this pattern (or until recently it did), with that I was wondering if you could update the reproduction and see if you apply this with Solid, otherwise, if not possible, we can leave the snippet in but won't reference any of them. Let me know, and we'll go from there.

Reply:
I'ven't figured out that scenario, I've just added it inside the solid sb sandbox repo ([here](https://github.com/webblocksapp/solidjs-sb-sandbox/blob/master/src/stories/ListStoryUncheckedChildren.stories.tsx)) and it's not supported. So I've removed the references of any of them in the docs.